ot-efr32.slce

```diff
@@ -0,0 +1,9 @@
+id: ot-efr32
+version: 0.0.1
+description: "ot-efr32 extension for Gecko SDK Suite"
+label: "Silicon Labs Matter"
+sdk:
+  id: gecko_sdk
+  version: 4.2.1
+component_path:
+  - path: slc/component
```

Comment:
Is `slc/component` directory exist already?

Reply:
It doesn't exist because git doesn't let you commit empty folders, but this is where ot-efr32 components will go. This isn't an issue though. Slc doesn't error out.